kloppy/domain/services/state_builder/__init__.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import replace
+
+from kloppy.domain import List, EventDataset
+
+from .base import StateBuilder
+from .lineup import LineupStateBuilder
+from .score import ScoreStateBuilder
+from .sequence import SequenceStateBuilder
+
+
+def create_state_builder(builder_key: str):
```

Comment:
Why this instead of just using the class as key? How do I add new state builders as a consumer of the library?

Reply:
Solved with a Registery.